docker-compose.yml

```diff
@@ -4,14 +4,20 @@ networks:
   todoapp-net:
 services:
   pythonapp:
-    image: todoapp:2.0.0
+    image: todoapp:${IMAGE_VERSION:-2.0.0}
     build:
       context: .
       dockerfile: Dockerfile
     ports:
       - "8080:8080"
     environment:
       - PYTHONUNBUFFERED=1
+      - ENGINE=${ENGINE:-mysql.connector.django}
+      - NAME=${NAME:-app_db}
+      - USER=${USER:-app_user}
+      - PASSWORD=${PASSWORD:-1234}
```

Comment:
Using a default password like `1234` is not secure. Consider using a more secure password or managing secrets through environment variables or a secrets management tool.